CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to the Project
+
+Hey there! Welcome to the project! Here's some things to keep in mind before you get started.
+
+# Questions
+
+If you have a question, and there isn't any specific Issue that the question is about, please post your questions in the [discussion](https://github.com/orgs/alveusgg/discussions).
+
+# Issues
+
+### Bug Reports:
+
+When reporting a bug, please open an issue following this format:
+
+- `Title`
+- `Expected Behavior`
+- `Current Behavior`
+- `Any Relevant Screenshots`
+- Go to [what's my browser](https://www.whatsmybrowser.org/), copy the link generated and paste it into the issue description. E.g. `https://www.whatsmybrowser.org/b/WTBQ3`
+- Add the `bug` label
+
+### Ideas / Enhancements
+
+When sharing an idea or enhancement open an issue in the following format:
+
+- `Title`
+- `Description` of the idea
+- `Justification` of why this idea would be beneficial
+- Add the `idea` or `enhancement` label.
```

Comment:
Instead of having this written out here, I think we'd be better off setting up some [issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) that prompt for these fields?